WooCommerce/Classes/ViewRelated/Orders/ProductListViewController.swift

```diff
@@ -114,6 +118,18 @@ private extension ProductListViewController {
         return items[indexPath.row]
     }
 
+    func lookUpProduct(by productID: Int) -> Product? {
+        guard let products = products else {
+            return nil
+        }
+
+        for product in products where product.productID == productID {
```

Comment:
I think the whole method could be also written just like this:

```
return products.filter({$0/productID == productID}).first
```

Reply:
Ah, thank you! I couldn't remember where I had wrote the last `.filter` change. This is so much cleaner.